draft-ietf-quic-transport.md

```diff
@@ -2405,6 +2433,20 @@ A receiver MAY use an autotuning mechanism to tune the size of the offset
 increment to advertise based on a roundtrip time estimate and the rate at which
 the receiving application consumes data, similar to common TCP implementations.
 
+### Stream Limit Increment
+
+As with flow control, this document leaves when and how many streams to make
+available to a peer via LIMIT_UPDATE to the implementation, but offers a few
+considerations. LIMIT_UPDATE frames constitute overhead, and therefore, sending
+a LIMIT_UPDATE with small offset increments is undesirable.  At the same time,
```

Comment:
The only case where this is a noticeable overhead is when each stream is only used to transfer a small number of bytes, e.g. in the often cited case of video streaming where every frame consumes a stream. For HTTP (and probably all other use cases) sending the 5 bytes consumed by the LIMIT_UPDATE frame will be negligible.